pkg/compute/init.go

```diff
@@ -90,6 +90,16 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 	text.Output(out, "Press ^C at any time to quit.")
 	text.Break(out)
 
+	files, err := os.ReadDir(".")
+	if err != nil {
+		return err
+	}
+
+	if len(files) > 0 {
+		text.Warning(out, "The current directory isn't empty. This might prevent the CLI from building your project if there are conflicting language configuration files.")
```

Comment:
```suggestion
		text.Warning(out, "The current directory isn't empty. Are you sure you want to initialize a Compute@Edge project in $PWD?")
```

Maybe something like this? `$PWD` would probably require calling `os.Getwd` i think?

Reply:
OK making changes locally and just wanted to share the prompt feedback...

**You don't want to continue...** (i.e. we short-circuit the execution)
<img width="1245" alt="Screenshot 2021-04-12 at 15 58 10" src="https://user-images.githubusercontent.com/180050/114415886-01169400-9ba8-11eb-8121-6296b1e16303.png">

**You don't type what we're expecting...** (i.e. we short-circuit the execution)
<img width="1333" alt="Screenshot 2021-04-12 at 15 58 17" src="https://user-images.githubusercontent.com/180050/114415978-18ee1800-9ba8-11eb-9cce-b23811fc366a.png">

**You want to continue...** (i.e. we continue with the execution flow)
<img width="1234" alt="Screenshot 2021-04-12 at 15 58 25" src="https://user-images.githubusercontent.com/180050/114416047-2efbd880-9ba8-11eb-86d0-4565bc114ce6.png">

Reply:
I love it!! :heart_eyes_cat: